spec/classes/nginx_spec.rb

```diff
@@ -159,7 +161,7 @@
           is_expected.to contain_apt__source('nginx').with(
             'location'   => "https://nginx.org/packages/#{operatingsystem.downcase}",
             'repos'      => 'nginx',
-            'key'        => '573BFD6B3D8FBC641079A6ABABF5BD827BD9BF62'
+            'key'        => { 'id' => '573BFD6B3D8FBC641079A6ABABF5BD827BD9BF62' }
```

Comment:
I think this was from another commit? Next time, rebase rather than merge master.